test/404s.test.js

```diff
@@ -443,3 +443,47 @@ test('setNotFoundHandler should not suppress duplicated routes checking', t => {
     t.ok(err)
   })
 })
+
+test('log debug for 404', t => {
+  t.plan(1)
+
+  const Writable = require('stream').Writable
+  class LogWritable extends Writable {
```

Comment:
Can we please lose the `class` syntax and just pass an object literal with a `_write` property?